### ISSUE TITLE
[bug] failed to create miner in localnet without sufficient balance 

### ISSUE THREAD

**Issue by urugang** [filecoin-project/go-filecoin#2733]
**Describe the bug**
1) account A send account B some coin.  
2) account A received transaction confirmation.  
3) account B did not receive transaction instantly, so account B still have no enough coin to create miner.  

**To Reproduce**

```
$ cd $GOPATH/src/github.com/filecoin-project/go-filecoin/tools/fast/bin/localnet
$ go build . && ./localnet  -miner-count=1 

10:58:08.564  INFO /tmp/local: RunCmd: /tmp/localnet989732221/0 [go-filecoin message send --value 100000 --from t1nhudgskmseowv7rp4e6scxsmlt3qoysvpn73tuy --gas-price 1 --gas-limit 300 t1m4m53jsucebf26n5d6cuw7avgfx7t2luhhc66dy --enc=json] process.go:191
10:58:08.593  INFO /tmp/local: RunCmd: /tmp/localnet989732221/0 [go-filecoin message wait zDPWYqFD81a4JkTSyjLiunBbLZcyF61J8z1Xg6YM8qLqnSdhf4ZH --enc=json] process.go:191
10:58:11.192  INFO /tmp/local: RunCmd: /tmp/localnet989732221/1 [go-filecoin miner create --gas-price 1 --gas-limit 300 10 500 --enc=json] process.go:191
failed series.CreateMinerWithAsk; filecoin command: [go-filecoin miner create --gas-price 1 --gas-limit 300 10 500 --enc=json], exited with non-zero exitcode: 1
10:58:11.232  INFO environmen: Teardown environment environment_memory_genesis.go:148
 ```

**Expected behavior**
wait a limited time span until enough coin to create miner actor.

**Screenshots**

**Version information**
`
$ git rev-parse HEAD  
5d3bd4cc0e9a27d750c2b4cf7ca0352581af76f7
`

**Comment by travisperson** [comment]
I think the issue you are running into is related to #2661, which was fixed in #2686.

Can you see if can repro your issue on the latest master branch?

**Comment by urugang** [comment]
yes, your pr works, and it is a better way.
